tests/jquery.flot.hover.Test.js

```diff
@@ -258,5 +258,81 @@ describe("flot hover plugin", function () {
 
             expect(getEntireCanvasData(canvas)).toContainPixelColor(rgba(10, 20, 30, 1));
         });
+
+        it('should call hooked function when canvas is hovered over', function() {
+            var spy = jasmine.createSpy('spy');
+
+            options.hooks = {findNearbyItems: [spy]};
+            plot = $.plot(placeholder, [ [ [0, 0], [2, 3], [10, 10] ] ], options);
+
+            var eventHolder = plot.getEventHolder(),
+                offset = plot.getPlotOffset(),
+                axisx = plot.getXAxes()[0],
+                axisy = plot.getYAxes()[0],
+                x = axisx.p2c(2) + offset.left,
+                y = axisy.p2c(3) + offset.top,
+                noButton = 0;
+
+            simulate.mouseMove(eventHolder, x, y, noButton);
+            jasmine.clock().tick(100);
+
+            expect(spy).toHaveBeenCalled();
+        });
+
+        it('should pass item returned from hook in items', function() {
+            var testItem = {distance: 5};
+            var hook = function(_0, _1, _2, _3, _4, _5, items) {
+                items.push(testItem);
+            }
+
+            options.hooks = {findNearbyItems: [hook]};
+            plot = $.plot(placeholder, [ [ [0, 0], [2, 3], [10, 10] ] ], options);
+
+            $(plot.getPlaceholder()).on('plothover', function(event, pos, item, items) {
+                var seenTestItem = false;
+                for (const i of items) {
+                    if (i === testItem) {
+                        seenTestItem = true;
+                    }
+                }
+                expect(seenTestItem).toBe(true);
+            });
+
+            var eventHolder = plot.getEventHolder(),
+                offset = plot.getPlotOffset(),
+                axisx = plot.getXAxes()[0],
+                axisy = plot.getYAxes()[0],
+                x = axisx.p2c(2) + offset.left,
+                y = axisy.p2c(3) + offset.top,
+                noButton = 0;
+
+            simulate.mouseMove(eventHolder, x, y, noButton);
+            jasmine.clock().tick(100);
+        })
+
+        it('should choose closest item from items returned by hooks', function() {
+            var distance = 5,
```

Comment:
This test doesn't plot any data, looks like it's relying on the plot from the previous test.

Reply:
Fixed.